sdk/onlineexperimentation/Azure.ResourceManager.OnlineExperimentation/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Other Changes
 
-## 1.0.0-beta.1 (2025-05-23)
```

Comment:
This has already released after https://github.com/Azure/azure-sdk-for-net/pull/49741 merged, so please don't change this.

Reply:
Reverted.